source/developers-guide/elasticsearch/index.md

```diff
@@ -1908,3 +1908,79 @@ class SalesSortingHandler implements HandlerInterface
 
 Like the other handlers, the `SalesSortingHandler` handles a `CriteriaPart`, so it implements the `Shopware\Bundle\SearchBundleES\HandlerInterface`.
 To sort the search result, the `ONGR\ElasticsearchDSL\Sort\Sort` class can be added to the provided `ONGR\ElasticsearchDSL\Search`.
+
+## Shopware 5.3 changes
+With Shopware 5.3 a new filter behavior was implemented which has to display only filters which can be combined with the current user conditions.
```

Comment:
Could you rephrase? Something like `A new filter behaviour was implemented with shopware 5.3, now filters are only displayed if they can be combined with the user conditions that are currently in effect.` sounds better imho.